src/components/chat/Chat.tsx

```diff
@@ -30,11 +31,11 @@ export function Chat() {
 
   // #region Derive isPending state from time between `started` and `next` events
   useWhileMounted(() =>
-    chatService.observe({
+    chatFx.observe({
       started() {
         setIsPending(true);
       },
-      next() {
+      response() {
```

Comment:
`next` is the RxJS term, but RxFx is about providing more mnemonic and memorable names for things (in my subjective opinion)